src/presence_analyzer/templates/presence_weekday.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html' %}
+
+{% block title %}
+    <title>Presence weekday</title>
+{% endblock %}
+
+{% block head %}
+    <script type="text/javascript">
+        google.load("visualization", "1", {packages:["corechart"], 'language': 'en'});
+
+        (function($) {
+            $(document).ready(function(){
+                var loading = $('#loading');
+                $.getJSON("/api/v1/users", function(result) {
+                    var dropdown = $("#user_id");
+                    $.each(result, function(item) {
+                        dropdown.append($("<option />").val(this.user_id).text(this.name));
+                    });
+                    dropdown.show();
+                    loading.hide();
+                });
+                $('#user_id').change(function(){
+                    var selectedUser = $("#user_id").val(),
+                        $chartDiv = $('#chart_div');
+
+                    if(selectedUser) {
+                        loading.show();
+                        $chartDiv.hide();
+                        $.getJSON("/api/v1/presence_weekday/" + selectedUser, function(result) {
+                            var data = google.visualization.arrayToDataTable(result),
+                                chart = new google.visualization.PieChart($chartDiv[0]),
+                                options = {};
+                            $chartDiv.show();
+                            loading.hide();
+                            chart.draw(data, options);
+                        });
+                    }
+                });
+            });
+        })(jQuery);
+    </script>
+{% endblock %}
+
+{% block content %}
+    <h2>Presence by weekday</h2>
+{% endblock %}
```

Comment:
Yeah, but this block extends `content` div. I tend to name blocks as their parents which they relate to. But I agree that here it should be named `{% block header %}`